### PR TITLE
Python Sample Client Updates

### PR DESCRIPTION
**Pull request by yunussandikci**
Hi @mdumandag,
According to your [suggestions](https://github.com/hazelcast/hazelcast-cloud-python-sample-client/pull/3#issuecomment-759504261), I did these changes. Can you please check?

Thanks

### CODE REVIEW

File: client.py [hunk 1]
```diff
@@ -10,10 +11,11 @@
 """
 
 logging.basicConfig(level=logging.INFO)
-
+HazelcastCloudDiscovery._CLOUD_URL_BASE = "YOUR_DISCOVERY_URL"
```
Comment (mdumandag) [inline client.py:14]: Will this (and its import) be visible to the user? Personally, I think that users should not see such internal details

File: requirements.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-hazelcast-python-client~=4.0.0
\ No newline at end of file
+hazelcast-python-client~=4.0.0
```
Comment (mdumandag) [inline requirements.txt]: Only the second line should be enough, we can delete `hazelcast-python-client~=4.0.0`

File: client.py (hunk 1 above)
Comment (yunussandikci) [reply client.py:14]: No, It won't be visible in Production

File: requirements.txt (hunk 2 above)
Comment (yunussandikci) [reply requirements.txt]: Done, thank you 👍

**Pull request merged**